testing/srt-test-multiplex.cpp

```diff
@@ -76,7 +76,7 @@ struct MediumPair
     bytevector initial_portion;
     string name;
 
-    MediumPair(unique_ptr<Source> s, unique_ptr<Target> t): src(move(s)), tar(move(t)) {}
+    MediumPair(unique_ptr<Source> s, unique_ptr<Target> t): src(std::move(s)), tar(std::move(t)) {}
```

Comment:
This and below: by some reason clang strictly requires that `move` be always used as `std::move` explicitly. Probably because this is a library support for a language feature and could be easily confused if the possibly local `move` name is in use.